NEWS

```diff
@@ -11,6 +11,44 @@
 ### Features:
 ### Bugfixes:
 
+## 1.16.0 (January 21, 2024)
+### Features:
+#### UCP
+* Multiple performance improvements
+* Multiple improvements in error and debug messages
+#### UCT
+* Improved intra-node collectives performance for small messages
+* Use NIM/MEMIC for SHMEM atomics in CX7
+#### RDMA CORE (IB, ROCE, etc.)
+* Added FLID based routing support
+* Added latency and min_zcopy configuration variables to ROCm-IPC
+* Optimized xpmem tag matching
+#### TCP
+* Filtered out bridge devices from lane selection
+#### GPU (CUDA, ROCM)
+* Fixed compilation flags to support ROCm 6.0
+* Handle memh with multiple registrations
+* Set parameters dependent on GPU type
+#### Java
+* Locked OS thread before setting CUDA device
+#### Build
+* Fixed JUCX package publishing, so it will include support for ARM
+* Maven publish without rebulding jar
+* Publish separate jar files per platform
+* Refactored GPG sign and publish of Jar files
+### Bugfixes:
+#### UCP
```

Comment:
Consider rephrasing some of these, see also examples in previous versions.
Should be "Fixed <error / wrong condition description>" + (optionally) "by ...."
rather than "Fixed X to do Y"